battleship-client/src/main/java/com/epam/solid/nie/client/communication/SocketClient.java

```diff
@@ -18,6 +18,7 @@
         this.ip = ip;
     }
 
+    @Override
     public boolean run() throws IOException {
```

Comment:
javadoc

Reply:
there is separate task for adding javadocs

Reply:
ok

Reply:
Why are there binary files here? DS_Store ones?